### PR TITLE
crowdin in-context support

### PR DESCRIPTION
**Pull request by camnicklaus**

### CODE REVIEW

File: addon/services/crowdin.js [hunk 1]
```diff
@@ -0,0 +1,34 @@
+import Service from '@ember/service';
+import injectScript from 'dashboard/utils/inject-script';
```
Comment (kiwiupover) [inline addon/services/crowdin.js]: why import from dashboard?

File: addon/services/crowdin.js [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+
+export default Service.extend({
+  setupScriptId: 'setup-incontext',
+  enableScriptId: 'enable-incontext',
+
+  setup: function(context) {
+    const setupScriptId = context.get('setupScriptId');
+    const text = "var _jipt = []; _jipt.push(['project', 'outdoorsy-translations']);";
```
Comment (kiwiupover) [inline addon/services/crowdin.js]: do we need this to be a config thing? `'outdoorsy-translations'`

File: addon/services/crowdin.js [hunk 3]
```diff
@@ -0,0 +1,34 @@
+import Service from '@ember/service';
+import injectScript from 'dashboard/utils/inject-script';
+
+export default Service.extend({
+  setupScriptId: 'setup-incontext',
```
Comment (kiwiupover) [inline addon/services/crowdin.js]: are these necessary ?

File: addon/services/crowdin.js [hunk 4]
```diff
@@ -0,0 +1,32 @@
...
+
+  init() {
+    this._super(...arguments);
+    this.config = getOwner(this).resolveRegistration('config:environment');
+  },
+
+  enableInContext() {
+    this._setup();
+    this._enable();
+    this.set('isInContextLoaded', true);
```
Comment (kiwiupover) [inline addon/services/crowdin.js]: this needs to be in the ` this._enable().then(()=> { this.set('isInContextLoaded', true)});`

Comment (camnicklaus) [reply addon/services/crowdin.js]: I had to move the `then` to inside the `_enable` method cause thats were the promise resolves.

**Pull request merged**